selinux-contexts.ks.in

```diff
@@ -1,13 +1,30 @@
 #test name: selinux-contexts
-%ksappend repos/default.ks
 
+%ksappend repos/default.ks
 %ksappend common/common_no_payload.ks
 %ksappend payload/default_packages.ks
 
-%post
+%pre
+
+printf '\x25post\n' > /usr/share/anaconda/post-scripts/95-kstest-selinux-contexts.ks
```

Comment:
Hmm, maybe it might be easier to just include the `80-setfilecons` file here.

Reply:
Hm, "locking" it to the file actually sounds useful... good idea.

Reply:
I meant something like this.
```

%include /usr/share/anaconda/post-scripts/95-kstest-selinux-contexts.ks

%post
# Do the test here.
%end
```

It means that we will the restore the context twice, that is not so nice, but the test will more readable.

Reply:
Ah, that would be better, yes. Let's do that.

Reply:
Done!